docs/docs/apis/userinterface.md

```diff
@@ -46,11 +46,17 @@ measureLayoutRelativeToWindow(component: React.Component<any, any>):
 measureLayoutRelativeToAncestor(component: React.Component<any, any>,
     ancestor: React.Component<any, any>): SyncTasks.Promise<LayoutInfo>;
 
-// Measures the dimension of the full window (or screen, in the case
+// Measures the dimension of window (based on specified root view id or 
+// defaults to main window or screen, in the case
 // of non-windowed platforms); the dimensions can also be obtained for any
 // view (including your app's top-level view) using the onLayout
 // callback
-measureWindow(): Types.Dimensions;
+measureWindow(rootViewId?: string): Types.Dimensions;
+
+// Measures the dimension of the specified root view window or defaults to
+// main window (or screen, in the case of non-windowed platforms);
+// Support measuring for platforms with multi window (UWP)
+measureRootViewWindow(rootViewId: string | undefined): Types.Dimensions;
```

Comment:
For consistency, this should be "rootViewId?", not a union type.

Reply:
I forgot to remove that part from doc